src/Svg.ts

```diff
@@ -201,7 +201,22 @@ export class SVGDrawing {
                 return mainCanvas.svg();
             });
     }
-
+    private inShape(x: number, y: number,pt: number,side: number): boolean {
```

Comment:
Do we need this function in the dashboard branch?

Reply:
Yes, it is required.